k8s-helm-http-wrapper/helm-server.js

```diff
@@ -1,13 +1,14 @@
 ﻿'use strict';
 const HelmWrapper = require("./helm-wrapper");
 var express = require('express');
-
 var router = express.Router();
+var helmWrapper = new HelmWrapper();
+
+helmWrapper.init();
```

Comment:
It looks like a work-around to solve the init problem.
Solving the single init issue should be taken care by the service class business logic, not by the server who consumes it